lib/src/commonMain/kotlin/com/seazon/feedme/lib/ai/gemini/GeminiApi.kt

```diff
@@ -4,107 +4,298 @@ import com.seazon.feedme.lib.network.HttpManager
 import com.seazon.feedme.lib.network.HttpMethod
 import com.seazon.feedme.lib.network.HttpUtils
 import com.seazon.feedme.lib.network.NameValuePair
+import com.seazon.feedme.lib.translation.google.GoogleTranslateApi
 import com.seazon.feedme.lib.utils.orZero
 import com.seazon.feedme.lib.utils.toJson
+import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 
-class GeminiApi {
+// ================== 接口定义 ==================
 
-    companion object {
-        const val URL = "https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-flash:generateContent"
+/**
+ * 统一翻译/摘要服务接口
```

Comment:
Please try to accurately express the function through class names and function names instead of adding comments. Only add comments when the logic is more complex.